### PR TITLE
WIP ~ Add two updatable RPM repositories

### PR DESCRIPTION
**Pull request by dberzano**

### CODE REVIEW

**Comment by dberzano** [comment]
@teo this is being tested now: please let me know if it works for you

**Comment by teo** [comment]
> @teo this is being tested now: please let me know if it works for you

Will do.

File: publish/aliPublish-updatable-rpms.conf [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        yum -v clean expire-cache
+        yum install -y alisw-%(package)s
+
+      To use the newly created package:
+
+        aliswmod enter %(package)s/%(version)s
+
+      You can find the full set of instructions (including repo configuration) here:
+
+        https://aliceo2group.github.io/binaries
```
Comment (Barthelemy) [inline publish/aliPublish-updatable-rpms.conf]: this is not the correct URL any more. It should read "https://aliceo2group.github.io/quickstart/binaries.html"

Comment (dberzano) [reply publish/aliPublish-updatable-rpms.conf]: Thanks for the heads-up; if this works, we should also update the documentation with instructions for all the three RPM repositories.

**Pull request merged**